src/Aspire.Hosting.Azure.AppContainers/AzureContainerAppsInfrastructure.cs

```diff
@@ -634,6 +634,13 @@ BicepValue<string> GetHostValue(string? prefix = null, string? suffix = null)
                     return (url, secretType);
                 }
 
+                if (value is ParameterResource param)
```

Comment:
Consider a comment saying that it's important that this is resolved before connection strings.

But if a test catches a regression then not required.